src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java

```diff
@@ -75,13 +80,26 @@ public void run() {
     public void start(
-            @Option(names = {"-c", "--config"}, paramLabel = "<file>", description = "Use <file> for Hazelcast configuration.") String configFilePath,
-            @Option(names = {"-cn", "--cluster-name"}, paramLabel = "<name>", description = "Use the specified cluster <name> " + "(default: 'dev').", defaultValue = "dev") String clusterName,
-            @Option(names = {"-p", "--port"}, paramLabel = "<port>", description = "Bind to the specified <port>. Please note that if the specified port is in use, " + "it will auto-increment to the first free port. (default: 5701)", defaultValue = "5701") String port,
-            @Option(names = {"-i", "--interface"}, paramLabel = "<interface>", description = "Bind to the specified <interface>" + " (default: bind to all interfaces).") String hzInterface,
-            @Option(names = {"-fg", "--foreground"}, description = "Run in the foreground.") boolean foreground,
-            @Option(names = {"-j", "--jar"}, paramLabel = "<path>", description = "Add <path> to Hazelcast class path.") String additionalClassPath,
-            @Option(names = {"-J", "--JAVA_OPTS"}, paramLabel = "<option>", split = ",", description = "Specify additional Java" + " <option> (Use ',' char to split multiple options).") List<String> javaOptions)
+            @Option(names = {"-c", "--config"}, paramLabel = "<file>", description = "Use <file> for Hazelcast configuration.")
+                    String configFilePath,
+            @Option(names = {"-cn", "--cluster-name"}, paramLabel = "<name>",
+                    description = "Use the specified cluster <name> " + "(default: 'dev').", defaultValue = "dev")
+                    String clusterName,
+            @Option(names = {"-p", "--port"}, paramLabel = "<port>",
+                    description = "Bind to the specified <port>. Please note that if the specified port is in use, "
+                            + "it will auto-increment to the first free port. (default: 5701)", defaultValue = "5701")
+                    String port,
+            @Option(names = {"-i", "--interface"}, paramLabel = "<interface>",
+                    description = "Bind to the specified <interface> (default: bind to all interfaces).")
+                    String hzInterface,
+            @Option(names = {"-fg", "--foreground"}, description = "Run in the foreground.")
+                    boolean foreground,
+            @Option(names = {"-j", "--jar"}, paramLabel = "<path>", split = ",", description = "Add <path> to Hazelcast "
+                    + "classpath (Use ',' to separate multiple paths).")
+                    String[] additionalClassPath,
+            @Option(names = {"-J", "--JAVA_OPTS"}, paramLabel = "<option>", split = ",",
+                    description = "Specify additional Java <option> (Use ',' to separate multiple options).")
+                    List<String> javaOptions)
```

Comment:
We agreed to keep all command line options & parameters in one single line, for ease of reading. I already added a suppression to the checkstyle about it, but if you would like to go with this way, that is ok 👍

Reply:
Thanks @alparslanavci  I see, I'm just using Hazelcast default code style, which I prefer. We may reconsider this before release.